spaces/domains.json

```
@@ -216,5 +216,6 @@
   "voting.smartdex.finance": "smartdex-voting.eth",
   "vote.moonpirate.finance": "moonpirate.eth",
   "vote.mai.finance" : "qidao.eth",
-  "dao.modefi.io" : "mod.eth"
+  "dao.modefi.io" : "mod.eth",
+  "vote.snapshot.org": "snowswap.eth"
```

You should use your own domain.
```suggestion
  "dao.modefi.io" : "mod.eth",
  "vote.snowswap.org": "snowswap.eth"
```